invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -284,6 +285,14 @@ def load_custom_fields():
     }
 
 
+def get_user_communities_memberships():
+    """Return current identity communities memberships."""
+    memberships = current_communities.service.members.config.record_cls.get_memberships(
```

Comment:
We are accessing data layer from views (resources). Wouldn't it make sense to abstract this such that the `record_cls` is not exposed outside of the service?

```
memberships =. current_communities.service.get_memberships()
```

Then the rest would happen in the communities service. I'd like to avoid breaking the encapsulation, anything that is in part of the service should be exposed over methods (i.e. config). Subservices is a thin line, so there could potentially be  `communities.service.members.get_membership()`.